config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-periodics.yaml

```diff
@@ -99,3 +99,27 @@ periodics:
       - --ginkgo-parallel=30
       - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
       - --timeout=180m
+
+# Run Kubernetes performance e2e tests with EKS prod build 1.11
+- interval: 1h
+  name: ci-kubernetes-e2e-aws-eks-1-11-scalability
+  labels:
+    preset-service-account: "true"
+    preset-kubernetes-e2e-aws-eks-common: "true"
+    preset-kubernetes-e2e-aws-eks-1-11: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+      imagePullPolicy: Always
+      args:
+      - --timeout=200
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-version-skew=false
+      - --deployment=eks
+      - --provider=eks
+      - --gce-ssh=
+      - --extract=ci/latest
```

Comment:
`ci/latest` has a 1.14-alpha build, I assume this means we are NOT testing Kubernetes at all then ...

Reply:
See my comment above.

One more thing - I think we should change this to ci/stable3 or something to make sure we're not running newer tests against older k/k release.

Reply:
changing this to stable3 makes more sense.

Reply:
Will change it soon.